File: ProjectTracking.UI/src/app/models/projecttask.ts
import { Employee } from "./employee";
import { UserStory } from "./userstory";

export class ProjectTask{
    projectTaskID?: string;
    assignedTo?: string;
    taskStartDate?: Date;
    taskEndDate?: Date;
    taskCompletion?: string;
    employeeID?: number;
    employee?: Employee;
    userStoryID?: number;
    userStory?: UserStory;
}

File: ProjectTracking-WebAPI/Data/Models/ProjectTask.cs
﻿using System;
namespace ProjectTracking_WebAPI.Models
{
    public class ProjectTask
    {
        public int ProjectTaskID { get; set; }

        public string AssignedTo { get; set; }

        public DateTime TaskStartDate { get; set; }

        public DateTime TaskEndDate { get; set; }

        public string TaskCompletion { get; set; }

        public int EmployeeID { get; set; }

        public Employee Employee { get; set; }

        public int UserStoryID { get; set; }

        public UserStory UserStory { get; set; }
    }
}

